src/doc/src/reference/config.md

```diff
@@ -1048,9 +1048,9 @@ If none of `branch`, `tag`, or `rev` is set, defaults to the `master` branch.
 #### `[target]`
 
 The `[target]` table is used for specifying settings for specific platform
-targets. It consists of a sub-table which is either a platform triple or a
-[`cfg()` expression]. The given values will be used if the target platform
-matches either the `<triple>` value or the `<cfg>` expression.
+targets. It consists of a sub-table which is either a [platform triple][target triple] 
+or a [`cfg()` expression]. The given values will be used if the target platform
+matches either the [`<triple>`] value or the `<cfg>` expression.
```

Comment:
Same here. We probably don't need a link for `<triple>` to avoid too many noise. And under this section, the meaning of `<triple>` is quite clear. No need to repeat the link.